src/module/interactions.ts

```diff
@@ -0,0 +1,44 @@
+import { serveTLS } from "https://deno.land/std@0.80.0/http/server.ts";
+
+// TODO: https://github.com/discord/discord-api-docs/pull/2295/files#diff-07978ef9d619062fb1afbb70ea065124531246743444f4354cc9e6c8879f8780R256
+
+// 1. Your endpoint must be prepared to ACK a `PING` message
+// 2. Your endpoint must be set up to properly handle signature headers--more on that in [Security and Authorization]
+
+const serverOptions = {
+  hostname: "localhost",
+  // Made of our favorite numbers for the default port
+  port: 0704,
```

Comment:
I do not think ports can start with 0

Reply:
Its just a placeholder waiting for ayntee favorite number

Reply:
`1729` :joy: